include/sys/zio.h

```diff
@@ -421,7 +421,12 @@ struct zio {
 	uint64_t	io_offset;
 	hrtime_t	io_timestamp;	/* submitted at */
 	hrtime_t	io_delta;	/* vdev queue service delta */
-	uint64_t	io_delay;	/* vdev disk service delta (ticks) */
+	hrtime_t	io_delay;	/* Device access time (disk or */
+					/* file). */
+	hrtime_t	io_disk_delay;	/* Same as io_delay, but for disks */
+					/* only.  Used for printing the */
+					/* delay in error messages in */
+					/* vdev_disk.c */
```

Comment:
Let's get rid of `io_disk_delay` entirely, with your updated infrastructure it doesn't retain any real value and will only serve as a future point of confusion.